vividus-bdd-engine/src/main/java/org/vividus/bdd/model/jbehave/Story.java

```diff
@@ -53,4 +60,60 @@ public void setScenarios(List<Scenario> scenarios)
     {
         this.scenarios = scenarios;
     }
+
+    /**
+     * Get unique scenarios.
+     *
+     * All iterations over story-level and scenario-level ExamplesTables are folded into unique scenario instances
+     *
+     * @return list of values with {@link Scenario} type
+     */
+    public List<Scenario> getFoldedScenarios()
+    {
+        if (lifecycle == null || lifecycle.getParameters() == null)
```

Comment:
boolean org.apache.commons.lang3.ObjectUtils.anyNull(Object... values) ?

Reply:
it would be great if this object util takes suppliers as args, otherwise it will be NPE if lifecycle is null